src/backend/aspen/app/views/api_utils.py

```diff
@@ -8,6 +8,13 @@
 from aspen.database.models.usergroup import User
 
 
+def filter_usergroup_dict(
+    unfiltered_dict: Mapping[str, Union[str, bool]],
+    fields_to_keep: Iterable[Union[str, bool]],
```

Comment:
```suggestion
    fields_to_keep: Collection[str],
```

Reply:
what's the main difference between iterable and collection? They seem the same to me

Reply:
Collection supports the `in` operation.

Reply:
oh interesting that makes sense